<launch>
  <node name="rgb_to_hsv" pkg="sensor_fusion" type="rgb_to_hsv.py" output="screen"/>
  <node name="keyboard_driver" pkg="sensor_fusion" type="publish_key.py" output="screen"/>
  <node name="color_twist" pkg="sensor_fusion" type="color_twist.py" output="screen"/>
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyACM0"/>
  </node>
</launch>
